keel/leads/models.py

```diff
@@ -16,8 +16,9 @@ class CustomerLead(TimeStampedModel):
     )
 
     lead_source = models.PositiveSmallIntegerField(verbose_name="Lead Source", choices=LEAD_SOURCE)
-    resolution = models.PositiveSmallIntegerField(verbose_name="Resolution", choices=RESOLUTION)
-    email = models.EmailField(verbose_name="Email", unique=True, blank=False, null=True, default=None)
+    resolution = models.PositiveSmallIntegerField(verbose_name="Resolution", null=True, choices=RESOLUTION)
```

Comment:
ADD a new option for resolution
IN PROGRESS 
WHich would be default option when creating a lead